### PR TITLE
Users/Docker_Image.rst: Fix copyright violation

### PR DESCRIPTION
**Pull request by acashmkj**
Fixed copyright violation in Users/Docker_Image.rst by referencing the
source and added refernce to CC-BY-SA 4.0 license 

Closes https://github.com/coala/documentation/issues/482

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by Makman2** [comment]
Please use `Closes` instead of `Fixes`, the issue is not a bug.

File: Users/Docker_Image.rst [hunk 1]
```diff
@@ -1,6 +1,8 @@
 What is Docker?
 ===============
 
+According to `opensource.com <https://opensource.com/resources/what-docker>`__:
```
Comment (Makman2) [inline Users/Docker_Image.rst]: Hmm that reads bad as a first sentence. Rather I would make some kind of citation at the end^^

Also:
> The layout needs to be very clear that it is an unmodified copy only, and no editing of the text is desirable.

Not sure that this intention gets to the reader here^^

**Comment by Makman2** [comment]
unack 684e22a

File: Users/Docker_Image.rst (hunk 1 above)
Comment (acashmkj) [reply Users/Docker_Image.rst]: Made the requested changes

File: Users/Docker_Image.rst [hunk 2]
```diff
@@ -3,19 +3,21 @@ What is Docker?
...
+and testing the code.
+
+In a way, Docker is a bit like a virtual machine. But unlike a virtual
+machine, rather than creating a whole virtual operating system, Docker
+allows applications to use the same Linux kernel as the system that
+they're running on and only requires applications be shipped with things
+not already running on the host computer. This gives a significant
+performance boost and reduces the size of the application.
+
+Source for this information is `opensource.com <https://opensource.com/resources/what-docker>`__.
```
Comment (Makman2) [inline Users/Docker_Image.rst]: Maybe just `Cited from ...`

**Comment by Makman2** [comment]
Commit message: Please add a tag (`Docker_Image.rst` or `Users/Docker_Image.rst`) with a verb describing what you've done (E.g. "Fix copyright violation" or "Cite correctly" etc etc)

Also add please a commit body describing why the text has changed (to make a full cite without any differences).

**Comment by Makman2** [comment]
unack a263ecc

**Comment by acashmkj** [comment]
@jayvdb added that it is used under CC-BY-SA 4.0 along with the link.

**Comment by Makman2** [comment]
☝️ https://github.com/coala/documentation/pull/494#issuecomment-344690055

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
ack ca85ea3ed7f0704e950be4aead6d74838ca0537c

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@gitmate-bot ff

**Pull request merged**